ml-algorithms/src/main/java/org/opensearch/ml/engine/utils/FileUtils.java

```diff
@@ -45,16 +45,16 @@ public class FileUtils {
      * @throws IOException
      */
     public static List<String> splitFileIntoChunks(File file, Path outputPath, int chunkSize) throws IOException {
-        int fileSize = (int) file.length();
+        long fileSize = file.length();
```

Comment:
Thanks for fixing this. Can you add some unit test ?

Reply:
(1) There are no tests for this class.
(2) The test would require writing 2GB file on disk, do you think it's acceptable?